test/integration_test/migration_test.go

```diff
@@ -85,6 +85,7 @@ func testMigration(t *testing.T) {
 	t.Run("operatorMigrationMongoTest", operatorMigrationMongoTest)
 	t.Run("operatorMigrationRabbitmqTest", operatorMigrationRabbitmqTest)
 	t.Run("bidirectionalClusterPairTest", bidirectionalClusterPairTest)
+	t.Run("serviceAccountUpdate", serviceAccountUpdate)
```

Comment:
nit.. maybe we should name this serviceAndServiceAccount (or something similar. Someone lookin at the name wouldn't know we are doing service update tests in it too.

Reply:
fixed.